src_py/apiServer/transmitter.py

```diff
@@ -57,24 +57,19 @@ def updateCSV(self, currentPhase): # currentPhase is either "Training", "Predict
         for root, dirnames, filenames in os.walk(globe.INPUT_DATA_PATH):
             for filename in filenames:
                 if filename == f"{globe.experiment_flow_global.expFlow['CSV path']}_{currentPhase.lower()}.csv":
-                    csvfile = open(os.path.join(root, filename), 'r').readlines()
+                    with open(os.path.join(root, filename), 'r') as f:
```

Comment:
Try to avoid naming a variable with a single char.
It can lead to variable shadowing cases.

Reply:
changed to fuller name